tests/integration/targets/vmware_ops_provision_vm_test/tasks/get_vm_info.yml

```diff
@@ -8,6 +8,10 @@
     vm_name: "{{ vm_name }}"
   register: vm_info
 
+- name: debug vm_info
```

Comment:
is the debug task intend to be here?

Reply:
nope, I'll remove it.
I'm going to add another module to get info about whether the guest is a template